config/routes.rb

```diff
@@ -33,7 +33,7 @@
   resources :core_induction_programmes, path: "core-induction-programmes", only: %i[show index], as: "cip"
   get "download-export", to: "core_induction_programmes#download_export", as: :download_export
 
-  resources :years, controller: "core_induction_programmes/years", only: %i[show edit update] do
+  resources :years, controller: "core_induction_programmes/years", only: %i[edit update] do
```

Comment:
Does it make sense for edit and update to still exist once show has gone?

Reply:
Yes, because we still display the year content on the programme page - and we still want it to be editable.

That's why I added https://github.com/DFE-Digital/ecf-engage-and-learn/pull/39/files#diff-c3259a261a0af47899b0204532c0d9993803fcad09f2b6cfc161a272c2f2b1a8R9-R11